# T-800's
# Preliminary Report

- Thomas J Messier
- Nick Nobrega
- Moses Kuzu
- Jocelyn Keller
- Ian Busch


# Abstract:
The T-800's preliminary report on reducing energy use in the Sonoma State Dorms consists of our mitigation technique, estimations on energy savings, and a baseline estimation of energy use.  Data provided is of our baseline estimation of energy use in the dorms.  Unfortunately, we are not able to obtain any actual data record of realtime energy use in the dorms at this time.  In order to obtain a baseline estimation, the group devised a survey to obtain a database of the average electrical appliances found in a dorm, as well as the average times these appliances are used in hours.  Our baseline of individual energy use 1 kwh per person.  Our bulk energy use per dorm unit accounts for items used by all residents 31.5 Kwh; not including HVAC. 

In [1]:
import pandas as pd
pd.read_csv('Energy_Use_1.csv')

Appliance  Avg. Wattage\n(Watts)\n  Avg.Time Use\n(Hours)\n  \
0    Microwave                   1200.0                     0.16   
1       Laptop                     60.0                     3.00   
2   Cell Phone                      5.0                     3.00   
3           TV                     80.0                     2.50   
4  Game System                    100.0                     2.00   
5      Toaster                   1200.0                     0.03   
6       Other                     100.0                     0.16   
7        Total                      NaN                      NaN   

   Kwh\n(Energy)\nPer Day\n  
0                     0.192  
1                     0.180  
2                     0.015  
3                     0.200  
4                     0.200  
5                     0.036  
6                     0.176  
7                     0.999

In [2]:
import pandas as pd
pd.read_csv('Energy_Use_2.csv')

Kwh Per \nResident    0.999
0                Residents per\ndorm\n   4.5000
1                       Units Per Dorm   7.0000
2  Total Kwh \nPer Dorm\n(Aggregate)\n  31.4685

In [3]:
import pandas as pd
pd.read_csv('Energy_Use_3.csv')

Appliance  Avg. Wattage\n(Watts)\n  Avg.Time Use\n(Hours)\n  \
0        Fridge                    725.0                     6.00   
1   Wifi Router                     30.0                    24.00   
2  Coffee Maker                   1000.0                     0.16   
3          HVAC                      NaN                      NaN   
4         Total                      NaN                      NaN   

   Kwh\n(Energy)\nPer Day\n  
0                      4.35  
1                      0.72  
2                      0.16  
3                       NaN  
4                      5.23

Table above accounts for energy used by bulk appliances that every unit has installed and are used by every resident in the unit

In [4]:
Dorm_Units = 7
Total_Kwh_Bulk = 5.23
Dorm_Units * Total_Kwh_Bulk

36.61

In [5]:
Energy_Use_Residents = 31.4685
Energy_Use_Bulk = 36.61
Energy_Use_Residents + Energy_Use_Bulk 

68.07849999999999

- We estimate 68.08 Kwh are used per day running/using household appliances and personal electronic items in a typical Tuscany dorm building with 7 units per building
- We can use this amount, and a future HVAC electrical use estimation subtracted against a cumulative Kwh amount used per day in the same dorm to account for energy used just in lighting 
- We are in the process of obtaining access to electrical meters and HVAC equipment in the Tuscany dorms in order to obtain accurate, real time cumulative Kwh readings as well as equipment ratings to account for electricity use when heating the building during times of the year when it is necessary
- A baseline of Energy use was needed in order to obtain any data for our preliminary calculations and will be updated when further information is available

# Estimation Summary:

For our mitigation technique of installing “Watt-Stopper” Occupancy Light Switches instead of manual on/off currently installed in Sonoma State University dorms, we have estimated an annual potential savings total of 166 dollars. "Watt-Stopper” is the brand of switch installed in other buildings throughout Sonoma State University (SSU), and would be the only brand SSU housing staff would be able to install.  An accurate model of switch currently installed has not been determined.  In order to make an estimation of payback, we researched through google.com and found an average cost of 59 dollars per “Watt-Stopper” switch.  We have determined a max of 4 switches can be installed per dorm unit.  A total cost of 1,652 dollars per 7 dorm units, 28 total switches installed.  If this estimation holds true, we determine the payback period to be 10 years.  This payback period is estimated using a basic Net Present Value (NPV) calculation without accounting for inflation.  We have also evaluated the potential of putting the same investment into a simple interest savings account at a discount rate of 2 percent.  After the same time period of 10 years, the investment would generate 2,013 dollars in a savings account.  As a group we feel this difference would not make our mitigation technique invalid.


# Estimation Spreadsheet Link

https://docs.google.com/a/sonoma.edu/spreadsheets/d/1T69NgdVlxPg3XQOcGWo7wuRkRmN_cW6f6z8rgYt_lJE/edit?usp=sharing